### DISCUSSION TITLE
Xbitinfo

### DISCUSSION THREAD

**Discussion by raybellwaves** [zarr-developers/zarr-python#1110] [General]
I recently came across https://github.com/observingClouds/xbitinfo
Hope you don't mind me dropping the link here as my (and anyone else bookmark).

I believe there is an issue somewhere by @rabernat on the use of the compression technique within the zarr framework but couldn't find it

**Comment by jakirkham** [comment]
Thanks Ray! 🙏

Issue ( https://github.com/zarr-developers/numcodecs/issues/298 ) was discussing adding an implementation based on the paper to Numcodecs. PR ( https://github.com/zarr-developers/numcodecs/pull/299 ) implemented this. It should be available starting Numcodecs 0.10.0 ( though there were a few wrinkles that only got fixed in 0.10.2; https://github.com/zarr-developers/numcodecs/pull/347 ).
